### PR TITLE
 Added code of daily Coding Exercise-22

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

**Review by waghmaresuyog — APPROVED** [review]

File: First/dailyExercises/checkArraysEqual.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package firstPackage.First.dailyExercises;
+
+public class checkArraysEqual {
```
Comment (prashantkawa) [inline First/dailyExercises/checkArraysEqual.java]: Class name should start with Capital letter

File: First/dailyExercises/checkArraysEqual.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    public static void main(String[] args) {
```
Comment (prashantkawa) [inline First/dailyExercises/checkArraysEqual.java]: add space between class and method

File: First/dailyExercises/checkArraysEqual.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        } else {
+            System.out.println("Arrays are not equal");
+        }
+    }
+}
+//OUTPUT:
+// "C:\Program Files\Java\jdk-17\bin\java.exe" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2023.1\lib\idea_rt.jar=51073:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2023.1\bin" -Dfile.encoding=UTF-8 -classpath "C:\Users\anushrees\Desktop\New folder\DemoRepo\First\out\production\First1" firstPackage.First.dailyExercises.checkArraysEqual
+//Arrays are not equal
+//
+//Process finished with exit code 0
```
Comment (prashantkawa) [inline First/dailyExercises/checkArraysEqual.java]: Add readme file for problem statement and result

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: First/dailyExercises/checkArraysEqual.java (hunk 1 above)
Comment (anushreeSonak) [reply First/dailyExercises/checkArraysEqual.java]: Done !

File: First/dailyExercises/checkArraysEqual.java (hunk 2 above)
Comment (anushreeSonak) [reply First/dailyExercises/checkArraysEqual.java]: Done !

File: First/dailyExercises/checkArraysEqual.java (hunk 3 above)
Comment (anushreeSonak) [reply First/dailyExercises/checkArraysEqual.java]: DONE !

**Review by prashantkawa — APPROVED** [review]

**Pull request merged**